<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeltaEMainWindow</class>
 <widget class="QMainWindow" name="DeltaEMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeltaEMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pBtn_Connect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_Adjust">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Adjust</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_Check">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_Run">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_massage">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Massage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_Color">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="logtextBrowser">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDeltaE_Tool_CVTE">
    <property name="title">
     <string>setting</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDeltaE_Tool_CVTE"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
